### PR TITLE
Change from json-rpc to grpc

### PR DESCRIPTION
**Pull request by DOBEN**
## Purpose

- JSON-RPC is deprecated and will be eventually removed from all our libraries. This PR migrates the eSealing dApp.

## Changes

- Replace using jsonRPC with gRPC

### CODE REVIEW

File: examples/eSealing/src/utils.ts [hunk 1]
```diff
@@ -25,10 +25,10 @@ export async function register(
             receiveName: `${E_SEALING_CONTRACT_NAME}.registerFile`,
             maxContractExecutionEnergy: 30000n,
         } as UpdateContractPayload,
```
Comment (bisgardo) [inline examples/eSealing/src/utils.ts:27]:
```suggestion
        },
```

File: examples/eSealing/src/eSealing.tsx [hunk 2]
```diff
@@ -155,8 +156,8 @@ export default function SEALING(props: WalletConnectionProps) {
 
     useEffect(() => {
         // View file record.
-        if (connection && account && fileHashHex !== '') {
-            withJsonRpcClient(connection, (rpcClient) => viewFile(rpcClient, fileHashHex))
+        if (grpcClient && account && fileHashHex !== '') {
```
Comment (bisgardo) [inline examples/eSealing/src/eSealing.tsx:159]: I know this it isn't part of this change, but should `account` be included here (and in the dependency list)? It's not being used inside.

**Review by bisgardo — APPROVED** [review]
LGTM

File: examples/eSealing/src/eSealing.tsx (hunk 2 above)
Comment (DOBEN) [reply examples/eSealing/src/eSealing.tsx:159]: I removed the `account` as a dependency. It is not needed here.

**Pull request merged**